hack/local-development/common/local-imagevector-overwrite

```diff
@@ -35,14 +42,13 @@ cleanup_imagevector_overwrite() {
 
 compute_imagevector_overwrite() {
   imagevector_overwrite="images: []"
-
-  VERSION="$(cat $(dirname $0)/../../VERSION)"
+  VERSION="$(cat $2/VERSION)"
   if [[ "$VERSION" != *"-dev"* ]]; then
     echo "$imagevector_overwrite"
     return
   fi
+  gardener_images_without_tag="$(yaml2json < "$2/charts/images.yaml" | jq -r --arg IMAGE_SOURCE_REPOSITORY $1 '.images | map(select(.sourceRepository == $IMAGE_SOURCE_REPOSITORY and .tag == null))')"
```

Comment:
```suggestion
  imagevector_images_without_tag="$(yaml2json < "$2/charts/images.yaml" | jq -r --arg IMAGE_SOURCE_REPOSITORY $1 '.images | map(select(.sourceRepository == $IMAGE_SOURCE_REPOSITORY and .tag == null))')"
```

Reply:
What if somebody doesn't have its `charts` directory at repo root? Maybe also let him provide the path to the `charts` directory?